code/html/index.html

```diff
@@ -182,9 +188,9 @@ <h2>Before using this device you have to change the default password for the use
                     <div class="footer">
                         &copy; 2016-2018<br />
                         Xose Pérez<br/>
-                        <a href="https://twitter.com/xoseperez" target="_blank">@xoseperez</a><br/>
-                        <a href="http://tinkerman.cat" target="_blank">http://tinkerman.cat</a><br/>
-                        <a href="https://github.com/xoseperez/espurna" target="_blank">ESPurna @ GitHub</a><br/>
+                        <a href="https://twitter.com/xoseperez" rel="noopener" target="_blank">@xoseperez</a><br/>
```

Comment:
This is because of linting errors i get trying to compile HTML with current gulp config.
```
[gulp-htmllint] /home/foxel/projects/espurna/code/html/index.html [191,25]: (E058) rel="noopener" required for links with target="blank"
```
I've read about those attacks using ```window.opener```. They are preatty tricky and assume the site you are referring is malicious. But still to satisfy this linting rule I've added those. 
Regarding ```noreferrer``` I dont think it's related.